vtex-apps/search-result
# Fix/price range mobile

#### What is the purpose of this pull request?

Make Price Range work on mobile

#### What problem is this solving?

Price Range wasn't working on mobile

#### How should this be manually tested?

[workspace](https://iaronaraujo--storecomponents.myvtex.com/apparel---accessories/clothing)
Open this workspace on mobile mode then on filters use the price range and check that it works

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/8443580/62150914-89baa700-b2d5-11e9-816f-988f4b2a145f.png)


#### Types of changes
